tests/cypress/integration/actions_users/issue_1810_login_logout.js

```diff
@@ -64,7 +74,12 @@ context('When clicking on the Logout button, get the user session closed.', () =
                 responce = await responce['headers']['set-cookie'];
                 const csrfToken = responce[0].match(/csrftoken=\w+/)[0].replace('csrftoken=', '');
                 const sessionId = responce[1].match(/sessionid=\w+/)[0].replace('sessionid=', '');
-                cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`)
+                cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`);
+                if (Cypress.browser.family !== 'chromium') {
+                    cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                        cy.contains('button', 'OK').click();
+                    });
+                }
```

Comment:
Can we put the code to cypress command instead of duplication?

Reply:
Sure. Reworked. Thanks.